index.html

```diff
@@ -0,0 +1,199 @@
+          <li>
+            <a href="" target="_blank"
+              ><i class="fa-brands fa-facebook-f"></i>
+              <span class="sr-only">Facebook</span></a
+            >
+          </li>
+          <li>
+            <a href="" target="_blank"
+              ><i class="fa-brands fa-twitter"></i>
+              <span class="sr-only">Twitter</span></a
+            >
+          </li>
+          <li><a href="#" class="disabled-link">English</a></li>
+          <li><a href="#" class="disabled-link">My Page</a></li>
+          <li><a href="#" class="disabled-link">Logout</a></li>
+        </ul>
+      </section>
+      <!-- Desktop Nav-->
+      <nav class="desktop-nav">
+        <ul class="inner-content">
+          <li>
+            <a href="index.html" class="nav-logo"
+              ><img src="./images/formula-1-logo.png" alt="F1 Logo" />
+            </a>
+          </li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#" class="disabled-link">Program</a></li>
+          <li><a href="#" class="disabled-link">Join</a></li>
+          <li><a href="#" class="disabled-link">Sponsor</a></li>
+          <li><a href="#" class="disabled-link">News</a></li>
+          <li><a href="#" class="disabled-link highlighted">F1 Sustainability</a></li>
+        </ul>
+      </nav>
+      <!-- Mobile Nav -->
+      <nav class="mobile-nav">
+        <ul class="mobile-header-menu">
+          <li class="menu">
+            <a href="#"
+              ><i
+                class="fa-solid fa-bars open-modal"
+                data-open="nav-modal"
+                id="mobile-menu"
+              ></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+      <div class="modal" id="nav-modal" data-animation="slideInOutLeft">
+        <div class="modal-dialog">
+          <button
+            type="button"
+            class="close-modal"
+            aria-label="close modal"
+            id="close-menu"
+            data-close
+          >
+            ✕
+          </button>
+          <ul class="modal-nav-links">
+            <li><a href="index.html" class="web-header-link">Home</a></li>
+            <li><a href="about.html" class="web-header-link">About</a></li>
+            <li><a href="" class="web-header-link">Program</a></li>
+            <li><a href="" class="web-header-link">Join</a></li>
+            <li><a href="" class="web-header-link">Sponsor</a></li>
+            <li><a href="" class="web-header-link">News</a></li>
+            <li><a href="" class="web-header-link">F1 Sustainability</a></li>
+          </ul>
+        </div>
+      </div>
+    </header>
+    <main>
+      <section class="hero" id="hero-content">
+        <div class="hero-inner-content inner-content">
+          <h1>"Hello, Formula One Fans"</h1>
+          <div class="greeting">
+            <p class="large-text">
+              Formula One <br />Global Summit <br />
+              <span>2022</span>
+            </p>
+          </div>
```

Comment:
- [ ] Kindly change the paragraph to h1 because it is the biggest text on the page and the h1 to h2 or h3. Keep shining 💪